### PR TITLE
Feature/us5288 printing labels

### PR DESCRIPTION
**Pull request by jpc11623**

### CODE REVIEW

File: SignInCheckIn/Printing.Utilities/Services/PrintNodeService.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+            _printingRestClient = printingRestClient;
+        }
+
+        public int SendPrintRequest(PrintRequestDto printRequestDto)
+        {
+            //var client = new RestClient(_printingUrl);
+
+            // printnode only asks for the api key in the username, no pw needed
+            _printingRestClient.Authenticator = new HttpBasicAuthenticator(_printingApiKey, String.Empty);
```
Comment (IngageKrizJ) [inline SignInCheckIn/Printing.Utilities/Services/PrintNodeService.cs]: Inject this

File: SignInCheckIn/SignInCheckIn/Unity.config [hunk 2]
```diff
@@ -53,6 +64,15 @@
       </constructor>
     </register>
 
+    <!-- Printing REST client -->
+    <register name="printingRestClient" type="RestSharp.IRestClient" mapTo="RestSharp.RestClient">
```
Comment (IngageKrizJ) [inline SignInCheckIn/SignInCheckIn/Unity.config]: You can inject the authenticator like [this example](https://github.com/crdschurch/crds-angular/blob/development/Gateway/crds-angular/Unity.config#L156) from crds-angular

File: SignInCheckIn/MinistryPlatform.Translation.Test/Repositories/ChildSigninRepositoryTest.cs.orig [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+                    ContactId = 1444,
+                    HouseholdId = otherHouseholdId.GetValueOrDefault(),
+                    HouseholdPositionId = 2,
+                    FirstName = "First2",
+                    LastName = "Last2",
+                    DateOfBirth = new DateTime()
+                }
+            };
+
+<<<<<<< HEAD
```
Comment (dtkocher) [inline SignInCheckIn/MinistryPlatform.Translation.Test/Repositories/ChildSigninRepositoryTest.cs.orig]: @jpc11623 same issue as above

File: SignInCheckIn/MinistryPlatform.Translation.Test/Repositories/ChildSigninRepositoryTest.cs.orig [hunk 4]
```diff
@@ -0,0 +1,466 @@
...
+            _applicationConfiguration.Setup(mocked => mocked.HouseHoldIdsThatCanCheckIn).Returns("3,4,2");
+            _ministryPlatformRestRepository.Setup(mocked => mocked.UsingAuthenticationToken("auth")).Returns(_ministryPlatformRestRepository.Object);
+            _ministryPlatformRestRepository.Setup(mocked => mocked.Search<MpParticipantDto>(GetChildParticpantsByPrimaryHouseholdFilter(primaryHouseholdId.GetValueOrDefault()), _primaryHouseChildParticipantColumns)).Returns(primaryChild);
+            _ministryPlatformRestRepository.Setup(mocked => mocked.SearchTable<MpParticipantDto>("Contact_Households", GetChildParticpantsByOtherHouseholdFilter(primaryHouseholdId.GetValueOrDefault()), _otherHouseChildParticipantColumns)).Returns(otherChild);
+            _ministryPlatformRestRepository.Setup(mocked => mocked.SearchTable<MpParticipantDto>("Group_Participants", GetChildParticpantsByGroupFilter("12,13"), _groupChildParticipantColumns)).Returns(children);
+            _ministryPlatformRestRepository.Setup(mocked => mocked.Search<MpEventGroupDto>($"Event_ID_Table.[Event_ID] = {eventDto.EventId}", _eventGroupColumns)).Returns(mpEventGroupDtos);
+
+<<<<<<< HEAD
```
Comment (dtkocher) [inline SignInCheckIn/MinistryPlatform.Translation.Test/Repositories/ChildSigninRepositoryTest.cs.orig]: @jpc11623 Same issue.

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/Interfaces/IChildSigninRepository.cs.orig [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using MinistryPlatform.Translation.Models.DTO;
+
+namespace MinistryPlatform.Translation.Repositories.Interfaces
+{
+    public interface IChildSigninRepository
+    {
+<<<<<<< HEAD
```
Comment (dtkocher) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/Interfaces/IChildSigninRepository.cs.orig]: @jpc11623 Same here

File: SignInCheckIn/SignInCheckIn.Tests/Services/ChildSigninServiceTest.cs.orig [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+                    ContactId = 1443,
+                    HouseholdId = primaryHouseholdId.GetValueOrDefault(),
+                    HouseholdPositionId = 2,
+                    FirstName = "First1",
+                    LastName = "Last1",
+                    DateOfBirth = new DateTime()
+                }
+            };
+
+<<<<<<< HEAD
```
Comment (dtkocher) [inline SignInCheckIn/SignInCheckIn.Tests/Services/ChildSigninServiceTest.cs.orig]: Same

File: SignInCheckIn/SignInCheckIn/Services/ChildSigninService.cs.orig [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+            _eventService = eventService;
+            _kioskRepository = kioskRepository;
+            _printingService = printingService;
+            _pdfEditor = pdfEditor;
+        }
+
+        public ParticipantEventMapDto GetChildrenAndEventByPhoneNumber(string phoneNumber, int siteId)
+        {
+            var eventDto = _eventService.GetCurrentEventForSite(siteId);
+<<<<<<< HEAD
```
Comment (dtkocher) [inline SignInCheckIn/SignInCheckIn/Services/ChildSigninService.cs.orig]: Same

**Comment by dtkocher** [comment]
👍

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/ContactRepository.cs [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+                "Household_ID_Table.Household_ID",
+                "Household_Position_ID_Table.Household_Position_ID",
+                "Household_ID_Table.Home_Phone",
+                "Mobile_Phone",
+                "Nickname",
+                "Last_Name"
+            };
+
+            var contacts = _ministryPlatformRestRepository.UsingAuthenticationToken(apiUserToken)
+                .Search<MpContactDto>($"[Household_ID]='{householdId}' AND Household_Position_ID IN (1, 7)", contactColumnList);
```
Comment (IngageKrizJ) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/ContactRepository.cs]: Can the 1 and 7 come from a config file?

**Pull request merged**